packages/vitest/src/runtime/workers/base.ts

```diff
@@ -52,6 +53,7 @@ export async function runBaseTests(method: 'run' | 'collect', state: WorkerGloba
     state,
     evaluatedModules: state.evaluatedModules,
     spyModule,
+    createImportMeta: createNodeImportMeta,
```

Comment:
I made it explicitly passed via `startVitestModuleRunner` since otherwise `createNodeImportMeta` would break `@vitest/web-worker` on vm since `module.register` is not supported there.